Danyal_Khan/surveyform/apps/survey_form_app/templates/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Survey Form</title>
+	</head>
+	<body>
+		
+		<DIV>
+			<h1>Survey Form LIVE</h1>
+			<form action="/surveys/process" method="POST">
+					{% csrf_token %}
+				<h3> Name:<input type="text" name="name"></h3>
+				<h3> Dojo Location:<select name="dojo">
+					<option>Dallas</option>
+					<option>Mountain View</option>
+					<option>Seattle</option>
+					<option>California</option>
+					</select></h3>
+				<h3> Favorite Language: <select name="language">
+					<option>Python</option>
+					<option>Ruby</option>
+					<option>Mean</option>
+					<option>C#</option>
+					<option>PsuedoCode</option>
+					</select></h3>
+				<h3> Comments:</h3><textarea rows="3" cols="12" name="comments"></textarea>
```

Comment:
while h3 tags work, it's not the best use for them.  Label tags would be better for these inside of a form